src/main/java/frc/robot/robot_manager/RobotCommands.java

```diff
@@ -85,11 +85,21 @@ public Command feedingCommand() {
         .andThen(robot.waitForState(RobotState.IDLE_NO_GP));
   }
 
+  public Command waitFeedingCommand() {
+    return Commands.runOnce(robot::waitFeedRequest, requirements)
+        .andThen(robot.waitForState(RobotState.IDLE_NO_GP));
```

Comment:
this condition doesn't really make sense. if we are doing a waiting feed state, why would we be in idle no gp?